### PR TITLE
feat: additional flag to force start manager-api

### PR DESCRIPTION
**Pull request by bisakhmondal**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
additional flag `-f` or `--force` to force start Manager API with different configuration maybe when a PID file is already existing.

**Related issues**

in continuation with #1814 
https://github.com/apache/apisix-dashboard/pull/1814#discussion_r627478821

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: api/internal/utils/pid.go [hunk 1]
```diff
@@ -25,9 +25,13 @@ import (
...
 // WritePID write pid to the given file path.
-func WritePID(filepath string) error {
-	if _, err := os.Stat(filepath); err == nil {
-		return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
+func WritePID(filepath string, forceStart bool) error {
+	if pid, err := ReadPID(filepath); err == nil {
+		if !forceStart {
+			return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
+		} else {
+			fmt.Printf("force starting new instance. another instance of Manager API is already running with pid %d\n", pid)
```
Comment (starsz) [inline api/internal/utils/pid.go:33]: `is already` => `maybe` 
Is this would be better? Since the old manager API maybe is not running.

Comment (bisakhmondal) [reply api/internal/utils/pid.go:33]: sorry for the late reply. Yeah kinda makes sense though. If due to some reason the pid file cleanup didn't take place which is obviously very unlikely. Yet `maybe` sounds good.

**Review by iamayushdas — APPROVED** [review]

**Comment by bisakhmondal** [comment]
Thanks, Mr. Das :)

**Review by nic-chen — APPROVED** [review]

**Review by iamayushdas — APPROVED** [review]
LGTM

**Comment by nic-chen** [comment]
@starsz  please have a look.

**Review by starsz — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**